packages/components/modules/comments/CommentCreate/index.tsx

```diff
@@ -75,15 +76,10 @@ let nextClientMutationId = 0
  */
 const CommentCreate = forwardRef<HTMLInputElement, CommentCreateProps>(
   (
-    {
-      targetObjectId,
-      profileId,
```

Comment:
we should also remove the `profileId` from  `CommentCreateProps` and from its storybook